### PR TITLE
chore(package): Add keywords to the package.json

### PR DESCRIPTION
**Pull request by petedv**
Add keywords to assist the npmjs search function

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+  "version": "1.0.7",
   "description": "Sentinel automated security testing framework",
   "main": "index.js",
   "repository": "https://github.com/nintexplatform/sentinel",
+  "keywords": [
+    "sentinel",
+    "security",
+    "test",
+    "testing",
+    "docker",
```
Comment (sai-kambaiyyagari) [inline package.json]: @petedv Nice to have these keywords as well `cucumber` `gherkin`

Comment (petedv) [reply package.json]: Yup. Updated

Comment (perryloh) [reply package.json]: What about automated and automation

Comment (perryloh) [reply package.json]: What about automated and automation

Comment (petedv) [reply package.json]: I'll add automated. The other is there.

Comment (petedv) [reply package.json]: Nvm. I see it was merged. :)

**Pull request merged**